### ISSUE TITLE
Can't find pane: sh

### ISSUE THREAD

**Issue by rothgar** [dennishafemann/tmux-cssh#15]
Running `tmux-cssh.sh -ns host1 host2 host3` will open a new session with host1 connected but the others do not connect. After disconnecting from the session I found this output in the terminal.

```
* Connecting 'ssh host1'
* Connecting 'ssh host2'
can't find pane: sh_1422386134
* Connecting 'ssh host3'
can't find pane: sh_1422386134
set option: synchronize-panes -> on
[exited]
```

I tried moving aside my .tmux.conf and .tmux files and trying again with the same result. I'm using tmux 1.6-3.

**Comment by dennishafemann** [comment]
Hey Justin, sorry, that it took a short while to answer...

It's not actually a "bug", but the circumstance that you're using "tmux-cssh.sh" as the script-name. Installing from the deb-package won't have this effect because, the script-name is just "tmux-cssh". I found out, that tmux itself is having some issues with some different/special characters in it's session name... In your case it's the dot  tmux-cssh(dot).sh ... Because tmux-cssh was using the basename for a kind of session-name-prefix... I fixed it with commit abe451e87984de61d2a8015efed35803b2c84bbb.

Let's me know if it works for you with the fix.

Thanks for opening the issue.

Best regards, Dennis
